packs/technic/gt_new_horizons.json

```diff
@@ -0,0 +1,225 @@
+    "adventurebackpack",
+    "ae2stuff",
+    "ae2wct",
+    "afsu",
+    "angermod",
+    "AppleCore",
+    "appliedenergistics2",
+    "aquacreeper",
+    "architecturecraft",
+    "Aroma1997Core",
+    "AromaBackup",
+    "asielib",
+    "Automagy",
+    "Avaritia",
+    "avaritiaddons",
+    "AWWayofTime",
+    "Backpack",
+    "bartworks",
+    "battlegear2",
+    "Baubles",
+    "bdlib",
+    "beebetteratbees",
+    "berriespp",
+    "BetterAchievements",
+    "betterbuilderswands",
+    "betterloadingscreen",
+    "betterquesting",
+    "BiblioCraft",
+    "BiblioWoodsBoP",
+    "BiblioWoodsForestry",
+    "BiblioWoodsNatura",
+    "BinnieCore",
+    "BiomesOPlenty",
+    "blocklimiter",
+    "BloodArsenal",
+    "bq_standard",
+    "BrandonsCore",
+    "bt",
+    "BuildCraft|Compat",
+    "BuildCraft|Core",
+    "BuildCraftOilTweak",
+    "CarpentersBlocks",
+    "catwalks",
+    "cb4bq",
+    "chisel",
+    "chiseltones",
+    "CodeChickenCore",
+    "CoFHCore",
+    "compactkineticgenerators",
+    "computronics",
+    "croploadcore",
+    "CustomMainMenu",
+    "defaultworldgenerator",
+    "detravscannermod",
+    "DraconicEvolution",
+    "dreamcraft",
+    "EMT",
+    "endercore",
+    "EnderIO",
+    "EnderStorage",
+    "EnderZoo",
+    "enhancedlootbags",
+    "enhancedportals",
+    "eternalsingularity",
+    "examplemod",
+    "extracells",
+    "ExtraUtilities",
+    "FastCraft",
+    "findit",
+    "FloodLights",
+    "ForbiddenMagic",
+    "Forestry",
+    "ForgeMultipart",
+    "ForgeRelocation",
+    "fw",
+    "gadomancy",
+    "galacticgreg",
+    "GalacticraftCore",
+    "GalacticraftMars",
+    "galaxyspace",
+    "gendustry",
+    "GraviSuite",
+    "gregtech_addon",
+    "gtneioreplugin",
+    "gtnhtcwands",
+    "gttweaker",
+    "HardcoreDarkness",
+    "HardcoreEnderExpansion",
+    "harvestcraft",
+    "harvestthenether",
+    "HelpFixer",
+    "hodgepodge",
+    "holoinventory",
+    "HungerOverhaul",
+    "IC2",
+    "ic2nei",
+    "IC2NuclearControl",
+    "ifu",
+    "IguanaTweaksTConstruct",
+    "InGameInfoXML",
+    "inventorytweaks",
+    "IronChest",
+    "ironchestminecarts",
+    "IronTank",
+    "itlt",
+    "JABBA",
+    "journeymap",
+    "kekztech",
+    "lootgames",
+    "LunatriusCore",
+    "MagicBees",
+    "malisiscore",
+    "malisisdoors",
+    "Mantle",
+    "MineTweaker3",
+    "miscutils",
+    "modtweaker2",
+    "MouseTweaks",
+    "MrTJPCoreMod",
+    "Natura",
+    "naturescompass",
+    "NEIAddons",
+    "neiintegration",
+    "NodalMechanics",
+    "NotEnoughItems",
+    "OpenBlocks",
+    "OpenComputers",
+    "openglasses",
+    "OpenMods",
+    "openmodularturrets",
+    "openprinter",
+    "opensecurity",
+    "overloadedarmorbar",
+    "p455w0rdlib",
+    "ProjectBlue",
+    "ProjRed|Compatibility",
+    "ProjRed|Core",
+    "ProjRed|Expansion",
+    "ProjRed|Exploration",
+    "ProjRed|Fabrication",
+    "ProjRed|Illumination",
+    "ProjRed|Transmission",
```

Comment:
This block will need to be changed to either: `projectred_4_7` or `projectred` depending on which one is in use.